packages/go_router/test/delegate_test.dart

```diff
@@ -179,33 +194,59 @@ void main() {
         );
       },
     );
+
+    testWidgets('gets the value provided from pop',
```

Comment:
I think we should move these test to go_router_test and test GoRouter api directly.

Reply:
Yeah, true, I was in doubt where to place it. I will make a change 👍